docs/reference/ml/df-analytics/apis/get-dfanalytics-stats.asciidoc

```diff
@@ -444,6 +444,11 @@ The memory usage status. May have one of the following values:
 * `hard_limit`: usage surpassed the configured memory limit.
 --
 
+`memory_reestimate_bytes`::::
```

Comment:
I think the fields in each section are supposed to be in alphabetical order.  That would put this one first within the collapsible section.

Reply:
Fixed